src/main/java/org/devcourse/voucher/controller/console/dto/Status.java

```diff
@@ -0,0 +1,10 @@
+package org.devcourse.voucher.controller.console.dto;
+
+public enum Status {
+    RUNNING,
+    FINISH;
+
+    public boolean isRunning() {
+        return !this.equals(FINISH);
```

Comment:
```suggestion
        return this.equals(RUNNING);
```
이 아닌 이유가 있을까요? 
현재는 Status가 추가 되면 코드를 수정해야할 것 같아서요.

Reply:
[refactor: 조건 변경](https://github.com/prgrms-be-devcourse/springboot-basic/pull/749/commits/f68b30818c851b4726ef443d194e2c50d99634d5) 반영